typespec-tests/src/main/java/com/cadl/naming/implementation/NamingOpsImpl.java

```diff
@@ -142,7 +142,7 @@ Response<BinaryData> getAnonymousSync(@HostParam("endpoint") String endpoint,
      *     name: String (Required)
      *     data (Required): {
      *         data (Required): {
-     *             kind_id: String (Optional)
+     *             &#64;data.kind: String (Optional)
```

Comment:
This one's strange. I didn't do this...

Reply:
This appears to be modified during format.
https://github.com/Azure/autorest.java/blob/main/postprocessor/src/main/java/com/azure/autorest/postprocessor/implementation/CodeFormatterUtil.java#L188

before
![image](https://github.com/Azure/autorest.java/assets/53292327/d0308f2f-8945-4e59-a982-09e99e41d4ba)

after
![image](https://github.com/Azure/autorest.java/assets/53292327/a2673dfb-a3b2-4673-8cc8-b4c5eadd9861)